.shippable.yml

```diff
@@ -77,6 +78,11 @@ build:
             fi;
             rm -f modified_tests.args modified_boards.args;
           fi;
+      - >
```

Comment:
you need to make the changes in scripts/ci/run_ci.sh now.

Reply:
Yep. I'm out of office today, I will (hopefully) align it tomorrow